webshop/templates/includes/cart/coupon_code.html

```diff
@@ -0,0 +1,32 @@
+{% if coupon_code %}
+<h6 class="my-2">Already applied coupon code:</h6>
+<div
+    class="cart-coupon-code flex gap-2 align-items-center bg-light w-fit rounded"
+>
+    <div class="p-2"> {{ coupon_code}} </div>
+    <button class="btn ml-4 bt-remove-coupon-code">
+        <span>
+            <svg
+                class="icon sm remove-cart-item-logo"
+                width="18"
+                height="18"
+                viewBox="0 0 18 18"
+                xmlns="http://www.w3.org/2000/svg"
+                id="icon-close"
+            >
+                <path
+                    fill-rule="evenodd"
+                    clip-rule="evenodd"
+                    d="M4.146 11.217a.5.5 0 1 0 .708.708l3.182-3.182 3.181 3.182a.5.5 0 1 0 .708-.708l-3.182-3.18 3.182-3.182a.5.5 0 1 0-.708-.708l-3.18 3.181-3.183-3.182a.5.5 0 0 0-.708.708l3.182 3.182-3.182 3.181z"
+                    stroke-width="0"
+                ></path>
+            </svg>
+        </span>
+    </button>
+</div>
+<div class="d-flex align-items-center text-muted my-1">
+    <hr class="flex-grow-1" />
+    <span class="px-2">OR</span>
```

Comment:
As a general convention, "transformations" should be through CSS. 

```suggestion
    <span class="px-2 text-uppercase">or</span>
```

Reply:
fixed at 6a83c1e3b7180739848f5eba19c52076c74c15aa